plugins/python/parser/src/scripts/cc_python_parser/main.py

```diff
@@ -0,0 +1,72 @@
+import os
+import json
+from copy import copy
+from json import JSONDecodeError
+from pathlib import PurePath, Path
+from typing import List, Final
+
+from cc_python_parser.common.metrics import metrics, PythonParserMetrics
+from cc_python_parser.parse_exception import ParseException
+from cc_python_parser.parser import Parser
+from cc_python_parser.persistence.persistence import ModelPersistence
+
+
+def read_config() -> List[str]:
+    """
+    Reads projects to parse from 'config.json', if it exists. Projects can be skipped with the "status" key:
+    {
+        "projects": [
+            {
+                "project": "project_path",
+                "status": true/false
+            }
+        ]
+    }
+    """
+
+    config_file_name: Final[str] = 'config.json'
+    project_roots = []
+    if Path(config_file_name).exists():
+        with open(config_file_name, 'r') as config_file:
+            try:
+                config = json.load(config_file)
+            except JSONDecodeError as e:
+                print(f"Invalid JSON file: config.json (line: {e.lineno}, column: {e.colno})")
+                return project_roots
+            except Exception as e:
+                print(f"Error during config.json decoding: {e}")
+                return project_roots
+            for project in config['projects']:
+                if project['status']:
+                    project_roots.append(project['project'])
+    return project_roots
+
+def main():
+    def directory_exception(path: PurePath) -> bool:
+        directory = os.path.basename(os.path.normpath(str(path)))
+        return directory.startswith('.') or directory == 'venv'
```

Comment:
These shouldn't be hardcoded in the source code. We should support a "skipfile" instead, where the user can provide a set of files and directories to skip. It is not necessarily true that a virtual environment is located in "venv" directory. I'm also not sure if we need to skip 3pp dependencies by default.